Ironmon-Tracker.lua

```diff
@@ -132,19 +132,36 @@ function Main.LoadNext()
 	client.SetSoundOn(false)
 	local romname = gameinfo.getromname()
 	client.closerom()
-
-	local rombasename = string.match(romname, '[^0-9]+')
-	local romnumber = tonumber(string.match(romname, '[0-9]+')) + 1
-	local nextromname = ""
-	if rombasename == nil then
-		nextromname = Settings.config.ROMS_FOLDER .. "/" .. romnumber .. ".gba"
-	else
-		rombasename = rombasename:gsub(" ", "_")
-		nextromname = Settings.config.ROMS_FOLDER .. "/" .. rombasename .. romnumber .. ".gba"
-		print(nextromname)
-	end
-
-	client.openrom(nextromname)
+  
+  -- Split the ROM name into its prefix and numerical values
+  local romprefix = string.match(romname, '[^0-9]+')
+  local romnumber = string.match(romname, '[0-9]+')
+  if romprefix == nil then romprefix = "" end
+  
+  -- Increment to the next ROM and determine its full file path
+  local nextromname = string.format(romprefix .. "%0" .. string.len(romnumber) .. "d", romnumber + 1)
+  local nextrompath = Settings.config.ROMS_FOLDER .. "\\" .. nextromname .. ".gba"
```

Comment:
You may notice that there was a change recently that helped people with Linux by using the `/` rather than `\\`. The `/` works in both Windows and Linux, while the `\\` excludes Linux. Can you swap this line and line 152 to use the `/` instead?